.github/workflows/start-ci.yml

```diff
@@ -0,0 +1,70 @@
+name: Label to staging
+
+on:
+  pull_request_target:
+    types: [labeled]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    if: github.event.label.name == 'start-ci'
+    steps:
+
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+
+      - name: Setup GCloud
+        uses: GoogleCloudPlatform/github-actions/setup-gcloud@master
+        with:
+          project_id: ${{ secrets.GCP_PROJECT_ID }}
+          service_account_email: ${{ secrets.GCP_SA_EMAIL }}
+          service_account_key: ${{ secrets.GCP_SA_KEY }}
+          export_default_credentials: true
+
+      - uses: actions/cache@v1
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-
+
+      - name: Install Dependencies
+        run: npm ci
+
+      - name: Run Gatsby Build
+        run: npm run build-ci
+        env:
+          PATH_PREFIX: ${{ github.event.pull_request.number }}/
+
+      - name: Run Storybook Build
+        run: npm run build-storybook
+        env:
+          PATH_PREFIX: ${{ github.event.pull_request.number }}/
+
+      - name: Upload to GCS
+        run: gsutil -m rsync -R public gs://staging.nodejs.dev/${{ github.event.pull_request.number }}/
+
+      - name: Comment on PR
+        uses: actions/github-script@0.9.0
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
+            github.issues.createComment({
+              issue_number: context.pull_request.number,
```

Comment:
OK, it looks like it doesn't like this :(

Reply:
Assuming `context` is equal to the Action variable `github.event`, you can try `context.number`.

Reply:
Thanks, can't hurt to try that first. I'll send a PR

Reply:
You can also add a console log to inspect what is available on context

Reply:
Thanks, I finally figured out the weird debug flow after for the actions. `-run: echo "${{ toJSON(github.pull_request) }}"`